src/js/tracks/video-track-enums.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file video-track-enums.js
+ */
+
+/**
+ * https://html.spec.whatwg.org/multipage/embedded-content.html#dom-videotrack-kind
+ *
+ * enum VideoTrackKind {
+ *   "alternative",
+ *   "captions",
+ *   "main",
+ *   "sign",
+ *   "subtitles",
+ *   "commentary",
+ *   "",
+ * };
+ */
+const VideoTrackKind = {
```

Comment:
Also, Video and Audio tracks are the same right? Moving it into one file for all 3 sounds like a good choice to reduce duplication.